packages/dds/merge-tree/src/test/obliterate.rangeExpansion.spec.ts

```diff
@@ -143,9 +127,56 @@ describe.skip("obliterate", () => {
 		},
 		expectedText: "hello",
 	});
+	describe("zero length", () => {
+		itCorrectlyObliterates({
+			title: "zero length obliterate at the start of the string",
+			action: (helper) => {
+				helper.insertText("A", 0, "hello world");
+				helper.processAllOps();
+
+				helper.obliterateRange(
+					"A",
+					{ pos: -1, side: Side.After },
+					{ pos: 0, side: Side.Before },
+				);
+				helper.insertText("B", 0, "more ");
+			},
+			expectedText: "hello world",
+		});
+		itCorrectlyObliterates({
+			title: "zero length obliterate in the middle of the string",
+			action: (helper) => {
+				helper.insertText("A", 0, "hello world");
+				helper.processAllOps();
+
+				helper.obliterateRange(
+					"A",
+					{ pos: 0, side: Side.After },
+					{ pos: 1, side: Side.Before },
+				);
+				helper.insertText("B", 1, "more ");
+			},
+			expectedText: "hello world",
+		});
+		itCorrectlyObliterates.skip({
```

Comment:
Add a comment here for why this is skipped (and if you have an ADO item for tracking add that in the comment as well)

Reply:
https://dev.azure.com/fluidframework/internal/_sprints/taskboard/Team%20Scarlett/internal/Team%20Scarlett%20-%202024.Mojito?workitem=17785